packages/framework/src/Model/Script/ScriptFacade.php

```diff
@@ -146,14 +148,14 @@ public function getAllPagesScriptCodes()
      */
     public function getOrderSentPageScriptCodesWithReplacedVariables(Order $order)
     {
-        $scripts = $this->scriptRepository->getScriptsByPlacement(Script::PLACEMENT_ORDER_SENT_PAGE);
-        $scriptCodes = [];
-
-        foreach ($scripts as $script) {
-            $scriptCodes[] = $this->replaceVariables($script->getCode(), $order);
-        }
-
-        return $scriptCodes;
+        $scriptCodes = $this->getScriptCodesByPlacement(Script::PLACEMENT_ORDER_SENT_PAGE);
+
+        return array_map(
```

Comment:
I am not realy fan of array_map function. Could we rather keep foreach here?

I know it is not a big deal, but foreach is more obvious to me (and probably to others too). Array_map is harder to read and do not have to much inpact on microoptimization.

E.g. it could be also used here but it is not https://github.com/shopsys/shopsys/pull/2086/files#diff-417f6bc61f2a1d726925f3c1e70085bfd50f3ff6a292c1c8a947be31bbecf201R228.

Reply:
okay, changed in b6f5a32bc

_btw. I see a difference between those two methods. In first the input is an array of strings and I need to pass every value through a function, but the result is just still an array of (modified) strings. 
In the second I have an array of objects, and I need just one property from the objects._